contents/teams/pipeline/objectives.mdx

```diff
@@ -1,3 +1,10 @@
 ## Goals: Q2 2024
 
-... coming soon ...
+1. Batch exports UX improvements, e.g. error notifications and UI rewamp
```

Comment:
Could you add an owner and a 'why' to each goal?

Reply:
Sure, happy to add a "why" for each goal.
Is there a reason we need owners in our public website (we have them internally and they might get shuffled around)?

Reply:
I'm not sure where I would go to find it apart from the website, so it's the easiest place for it.

Reply:
The plan is to create GH issues for each of the goals - there will be clear owners there, that would also be up to date with reality.